src/server/services/csvPipeline/success.js

```diff
@@ -12,7 +14,7 @@
  */
 function success(req, res, comment = '') {
 	res.status(200) // 200 OK
-	.send(`<h1>SUCCESS</h1>${comment}`);
+	.send(`<h1>${translate('status.success')}</h1>${comment}`);
```

Comment:
Do we need the `h1` tags here still? Or should we remove them?

Reply:
True!

Reply:
nice